### ISSUE TITLE
support of tidyr verbs as gather

### ISSUE THREAD

**Issue by KnutJaegersberg** [xiaodaigh/disk.frame#192]
That would be handy, too. For now we can reshape data using map, but it would feel more natural to do that in well known tidyr syntax. think of gather etc. Using map works i.e. when you want to use gather / melt from the data table package, but it is a bit less intuitive. Whereever logically possible (chunking could be a limitation), it would be nice if disk.frames behave like dataframes in the tidyverse.

**Comment by wlandau** [comment]
I would suggest the new `pivot_longer()` and `pivot_wider()` functions. I think people usually find them easier to remember and use.

**Comment by xiaodaigh** [comment]
Yeah. New features are always great. I am currently focused on getting code testing coverage to as close to 100% as possible and writing more article and focusing on documentation. 

That, unfortunately, means that new features will have to wait unless you want to have a crack at implementing something using `map`.

As a side note, I find `pivot_longer` also confusing...
